test/runtests.jl

```diff
@@ -1,12 +1,15 @@
 using Mocking
 
+using Plots
 using Test
 using TimeZones
 using TimeZones: PKG_DIR
 using TimeZones.TZData: ARCHIVE_DIR, TZSource, compile, build
 using Unicode
 
 Mocking.activate()
+unicodeplots()  # so don't need X11 to test plotting
```

Comment:
you should be able to set `ENV["GKS_WORKSTATION"] = 100` or something to avoid needing X11, not sure what the exact name was (you can probably find out from the GR docs).  Consider using something like VisualRegressionTests.jl for visual regression if you need it.